src/docs/product/sentry-basics/search/searchable-properties.mdx

```diff
@@ -28,15 +28,17 @@ You can search by event properties in the following [sentry.io](https://sentry.i
 
 When searching on event properties within the **Issues** page, the search will return any issue that has _one or more events_ matching the supplied event properties filter.
 
-### Metrics Properties
+In the case of transaction events, we also refer to these types of properties as _indexed event properties_ because they are transaction events for which we store all of the data. This isn't the case for [processed event properties](#processed-event-properties). **For error events, we currently store all data for all events sent to Sentry, and the concept of processed events doesn't apply.** Learn more about indexed and processed transaction events in our full [Sampling documentation](/product/sentry-basics/sampling/).
```

Comment:
```suggestion
In the case of transaction events, we also refer to these properties as _indexed event properties_ because they are based on transaction events for which we store all of the data. This isn't the case for [processed event properties](#processed-event-properties). **For error events, we currently store all data for all events sent to Sentry, and the concept of processed events doesn't apply.** Learn more about indexed and processed transaction events in our full [Sampling documentation](/product/sentry-basics/sampling/).
```